src/utils/environments.ts

```diff
@@ -0,0 +1,21 @@
+export const getEnvironment = (environmentString: string) => {
+  switch (environmentString) {
+    case "prod":
+      return {
+        url: undefined,
+        apiKey: process.env.PROD_API_KEY,
+      };
+    case "dev":
+      return {
+        url: "https://dev.v1.orchestrator.rhinestone.dev",
+        apiKey: process.env.DEV_API_KEY,
+      };
```

Comment:
In dev we will always be using the dev contracts here

Reply:
yep thats expected